apps/openssl.c

```diff
@@ -231,6 +233,63 @@ static void setup_trace(const char *str)
 }
 #endif /* OPENSSL_NO_TRACE */
 
+static OSSL_CALLBACK self_test_events;
+static char *self_test_corrupt_desc = NULL;
+static char *self_test_corrupt_type = NULL;
+static int self_test_log = 1;
+
+static int self_test_events(const OSSL_PARAM params[], void *arg)
+{
+    const OSSL_PARAM *p = NULL;
+    const char *phase = NULL, *type = NULL, *desc = NULL;
+    int ret = 0;
```

Comment:
The ret variable here doesn't actually do anything. It isn't referenced in anything except "goto err" (where 0 is returned) and set to 1 at the 'end:' label.

You could remove it, and just change the end of the function to be:
```end:
    return 1;
err:
    return 0;
```